layouts/partials/footer/footer.html

```diff
@@ -4,7 +4,7 @@
       <div class="col">
         {{ partial "logo/logo.html" (dict "id" "logo") . }}
         <nav>
-          <ul class="body-1">
```

Comment:
ok, maybe the name of this class was miss leading (as it comes from Material Design, where you've `body-1` and `body-2` available).